### DISCUSSION TITLE
[OP Stack Deployment] After deploying a OP Stack testnet successfully, I couldn't bridge eth

### DISCUSSION THREAD

**Discussion by qings2329** [ethereum-optimism/developers#366] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

After deploying a OP Stack testnet successfully, I couldn't bridge eth  by sending some to the L1StandardBridgeProxy address, as this tx https://sepolia.etherscan.io/tx/0x9ed9ba02909221b1d43ab9b9718d30dfba8feea2dd5d93f47fb2c61674f5d549

here is the deployments/getting-started/.deploy :

{
  "AddressManager": "0xe90Bf24438759b7862729679bd1eF69d95bCe01C",
  "DisputeGameFactory": "0x6A3E4aF0dC3E1Ed2DF3C098E568A8399F43B8CE5",
  "DisputeGameFactoryProxy": "0xB3CFa940b327b407213310430c77d2aF84035895",
  "L1CrossDomainMessenger": "0xe683B6FF29b2d09098253f931F5Bb9Acb527b816",
  "L1CrossDomainMessengerProxy": "0xce4AeE55dC1F17D1f87cf04715022789D340cB75",
  "L1ERC721Bridge": "0x3a5e452b6f7787F6D009C64f998EDED63B89Fa13",
  "L1ERC721BridgeProxy": "0x9DD4343ca88598C340230A181077Ed69AA5Ded43",
  "L1StandardBridge": "0x3d4965442F708c2309f1bc680c78982Ff0951dEc",
  "L1StandardBridgeProxy": "0xc45B597E025A054C5eA2AA3854056f6954898729",
  "L2OutputOracle": "0xe17d7E2850D99821B025EE19d520611c03379621",
  "L2OutputOracleProxy": "0x3F49B622C794d0b08001061e8Da4473FC48c3132",
  "Mips": "0x7eFc278F1E8eCf67Ca527890D44c1e3ed5db4532",
  "OptimismMintableERC20Factory": "0xB42697f1D0aE36A24533982071f713BDA49f2076",
  "OptimismMintableERC20FactoryProxy": "0x15be8f649Eb86902C86343395097b8A70093b16A",
  "OptimismPortal": "0xbFc7A66722b89E07b1831eF11A172E57C57Fb95B",
  "OptimismPortalProxy": "0xc46B1dF25460638DFB113b1224Df1A959cc1b84C",
  "PreimageOracle": "0x0868Aba516800a4B4941ce7F3c45CCf83F3A53aE",
  "ProtocolVersions": "0x62E781cac9281F8c819B7A65846495f41F098a30",
  "ProtocolVersionsProxy": "0x2074cD071a29ed363a679395a81066dB7F567EFe",
  "ProxyAdmin": "0x316f3FB8D22036D8cA5b2dB24f3796Afde5460B0",
  "SafeProxyFactory": "0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2",
  "SafeSingleton": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
  "SuperchainConfig": "0x2c325f7e4cd100720e5F631CB667f565a407B88e",
  "SuperchainConfigProxy": "0x73831575c13059e47C9367e8BD30da85d3eaC348",
  "SystemConfig": "0xc63f611006F7f96205B19cBdbaF9a5e6427388C4",
  "SystemConfigProxy": "0x98d634e47C037F0684F63C11fDDdA863BA62fdE1",
  "SystemOwnerSafe": "0xdE91c11DBFC7e20e4795887f6DBE53377cDe336e"
}

### Logs

some log of op-node 

INFO [04-25|19:06:27.251] sequencer successfully built a new block block=759da5..495cb0:623 time=1,714,037,182 txs=1
INFO [04-25|19:06:27.609] creating new block                       parent=759da5..495cb0:623 l1Origin=caa1e1..72f46b:5773503
ERROR[04-25|19:06:29.783] sequencer temporarily failed to start building new block err="temp: failed to fetch L1 block info and receipts: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
INFO [04-25|19:06:30.093] Advancing bq origin                      origin=b14258..4d1471:5773502 originBehind=false
WARN [04-25|19:06:30.093] tx in inbox with unauthorized submitter  addr=0x94133005fc9efEEB15C383244C58a989eFc71F04 hash=7ba063..a5c7d4 err=<nil>
WARN [04-25|19:06:30.093] tx in inbox with unauthorized submitter  addr=0x73F6FCd457E2C229230D3fe058f07F1E2d9Cb71B hash=21fd26..4f09fc err=<nil>
WARN [04-25|19:06:30.093] tx in inbox with unauthorized submitter  addr=0x5C1736106082Aa6B3eC66FC04408aEEb4739B66c hash=d0fad9..bcfea2 err=<nil>
WARN [04-25|19:06:30.093] tx in inbox with unauthorized submitter  addr=0x6751be96a14912b492B09182eb7E52b2520C2eeD hash=e4c273..749a96 err=<nil>
WARN [04-25|19:06:30.093] tx in inbox with unauthorized submitter  addr=0x87c0Da3a03218Be364b89b460c1376752C5CEC96 hash=ad3f4a..37a6f4 err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0x36c727A69F4c8b123E7A879E16b2eb79da357A3b hash=2cce07..86350b err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0xBDBf78E9E9ff6dbdcC091F31201C351446182B12 hash=323049..415076 err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0x0F82E82268FA5de5070A088e54eAbc2dec07D615 hash=6d7836..9874c1 err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0x95aCE797246c061a79Ee52263828C9a8A55b9f78 hash=600957..28adf4 err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0xA869aF2883DD6C881bC59594aDfB3e8b136321b4 hash=2f923d..b71900 err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0xD837232D3cb8cE6a54e0dc10BFAe3D304004284e hash=20adb5..c8f66b err=<nil>
WARN [04-25|19:06:30.094] tx in inbox with unauthorized submitter  addr=0x7fa359aE024667C8d61866fC4b3E96ed8D3C8657 hash=da0273..a5cea6 err=<nil>
WARN [04-25|19:06:32.577] Derivation process temporary error       attempts=1  err="temp: failed to fetch receipts of L1 block 0xcaa1e1600496b506d8701dd8e32e2d8f86c56320147de6e07bf24c025572f46b:5773503 (parent: 0xb142581e04581730a340f2217929cdb31f2947323f77b05f2bd63d519f4d1471:5773502) for L1 sysCfg update: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
INFO [04-25|19:06:32.883] creating new block                       parent=759da5..495cb0:623 l1Origin=caa1e1..72f46b:5773503
ERROR[04-25|19:06:35.280] sequencer temporarily failed to start building new block err="temp: failed to fetch L1 block info and receipts: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
WARN [04-25|19:06:36.895] Derivation process temporary error       attempts=2  err="temp: failed to fetch receipts of L1 block 0xcaa1e1600496b506d8701dd8e32e2d8f86c56320147de6e07bf24c025572f46b:5773503 (parent: 0xb142581e04581730a340f2217929cdb31f2947323f77b05f2bd63d519f4d1471:5773502) for L1 sysCfg update: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
INFO [04-25|19:06:37.317] creating new block                       parent=759da5..495cb0:623 l1Origin=caa1e1..72f46b:5773503
ERROR[04-25|19:06:38.009] sequencer temporarily failed to start building new block err="temp: failed to fetch L1 block info and receipts: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
INFO [04-25|19:06:39.919] creating new block                       parent=759da5..495cb0:623 l1Origin=caa1e1..72f46b:5773503
ERROR[04-25|19:06:44.429] sequencer temporarily failed to start building new block err="temp: failed to fetch L1 block info and receipts: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
ERROR[04-25|19:06:46.949] error getting latest header              err="Too Many Requests, Please apply an OnFinality API key or contact us to receive a higher rate limit"
WARN [04-25|19:06:48.911] Derivation process temporary error       attempts=3  err="temp: failed to fetch receipts of L1 block 0xcaa1e1600496b506d8701dd8e32e2d8f86c56320147de6e07bf24c025572f46b:5773503 (parent: 0xb142581e04581730a340f2217929cdb31f2947323f77b05f2bd63d519f4d1471:5773502) for L1 sysCfg update: failed batch-retrieval: json: cannot unmarshal object into Go value of type []*rpc.jsonrpcMessage"
INFO [04-25|19:06:49.174] creating new block                       parent=759da5..495cb0:623 l1Origin=caa1e1..72f46b:5773503
INFO [04-25|19:06:49.575] sequencer started building new block     payload_id=0x0210c6b6eb120d27 l2_parent_block=759da5..495cb0:623 l2_parent_block_time=1,714,037,182
INFO [04-25|19:06:49.684] inserted block                           hash=d63c1e..12ea66 number=624 state_root=636f33..a022c1 timestamp=1,714,037,184 parent=759da5..495cb0     prev_randao=7d8f68..cce17e fee_recipient=0x4200000000000000000000000000000000000011 txs=1 update_safe=false
INFO [04-25|19:06:49.684] sequencer successfully built a new block block=d63c1e..12ea66:624 time=1,714,037,184 txs=1
INFO [04-25|19:06:50.263] creating new block                       parent=d63c1e..12ea66:624 l1Origin=caa1e1..72f46b:5773503
INFO [04-25|19:06:50.288] sequencer started building new block     payload_id=0x0248d048d9ef6bd1 l2_parent_block=d63c1e..12ea66:624 l2_parent_block_time=1,714,037,184
INFO [04-25|19:06:50.397] inserted block                           hash=bea08f..b5e6fd number=625 state_root=c430c2..8ed31c timestamp=1,714,037,186 parent=d63c1e..12ea66     prev_randao=7d8f68..cce17e fee_recipient=0x4200000000000000000000000000000000000011 txs=1 update_safe=false
INFO [04-25|19:06:50.397] sequencer successfully built a new block block=bea08f..b5e6fd:625 time=1,714,037,186 txs=1
INFO [04-25|19:06:53.534] creating new block                       parent=bea08f..b5e6fd:625 l1Origin=caa1e1..72f46b:5773503

**Comment by opfocus** [comment]
Usually this requires waiting for a while. https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-8570927

**Reply by yohanelly95** [reply]
how do we reduce the time taken? Or which code should we go through to see how this process works for deposits? I mean the op golang code, the contracts are fine since they emit an event

**Reply by opfocus** [reply]
Deposits are very fast and this is not caused by the deposit.

**Reply by yohanelly95** [reply]
Yes a few minutes from L1 to L2. What if the L1 block time is less than 12 seconds though, it does not work as intended then. Takes much longer around ~2 hours to bridge. What gives ?

**Reply by opfocus** [reply]
https://specs.optimism.io/protocol/deposits.html   What's here is probably highly relevant, but I don't know much about the code

**Reply by yohanelly95** [reply]
this got resolved, not sure why but I reran op-stack using a higher hardware config for my vm and it worked as expected. (16 GB works fine, got errors with 8GB hardware config)

**Comment by Chomtana** [comment]
You should send ETH to OptimismPortalProxy.

![image](https://github.com/ethereum-optimism/developers/assets/4103490/3ea38fd5-de6a-4052-9e4d-32f77edb05cc)

However, it should also be good to send to L1StandardBridgeProxy as the event is emitted. But I haven't tested that way.

**Reply by sbvegan** [reply]
ETH to the OptimismPortal is usually fine. When using the bridges/cross domain messengers you spend more gas but you get replay protection. So its advised for people to bridge through there, so if their transaction fails, they can try again. If the transaction fails going to the OptimismPortal, the tokens will get stuck and be unretrievable (without some smart contract surgery).
